geomesa-tools/README.md

```diff
@@ -153,30 +153,32 @@ To create a new feature on a specified catalog table, use the `create` command.
 #### Example command:
     geomesa create -u username -p password -c test_create -i instname -z zoo1,zoo2,zoo3 -fn testing -s fid:String:index=true,dtg:Date,geom:Point:srid=4326 -dt dtg
 
-### delete
-To delete a feature on a specified catalog table, use the `delete` command.
+### removeschema
```

Comment:
Following the lead of the feature => type discussion, is 'schema' clear to users who are not familiar with GeoTools?